papis/document.py

```diff
@@ -6,10 +6,7 @@
 from typing import (
     List, Dict, Any, Optional, Union, NamedTuple, Callable, Tuple)
 
-try:
-    from typing import TypedDict  # Python 3.8+
-except ImportError:
-    from typing_extensions import TypedDict
+from typing_extensions import TypedDict
```

Comment:
For some reason `mypy` on <3.8 was complaining about these
https://github.com/papis/papis/runs/8133303972?check_suite_focus=true
so went with just importing the `typing_extensions` version.